src/plugins/github/relationalView.js

```diff
@@ -176,6 +176,8 @@ export class RelationalView {
         return this.comment(address);
       case "USERLIKE":
         return this.userlike(address);
+      case "COMMIT":
+        throw new Error("Commit entity not yet supported");
```

Comment:
Would prefer to return `null` here, as clients could conceivably request
the endpoint of a PR’s merged-as edge.